workflow/scripts/get-reads.py

```
@@ -0,0 +1,39 @@
+
+
+def aln_to_fq(aln):
+    return dnaio.SequenceRecord(
+        name=aln.query_name,
+        sequence=aln.get_forward_sequence(),
+        quality=aln.query_qualities,
+    )
+
+
+limit = snakemake.params.limit
+bam = pysam.AlignmentFile(snakemake.params.bam_url)
```

Ruff (0.11.9)

Undefined name `snakemake`

(F821)